custom_components/test/fuzzing/value_pools/test_vp_on_helpers.py

```diff
@@ -135,12 +134,10 @@ def test_get_miniserver_type() -> None:
     assert result["failed_tests"] == 0
 
 
-
 @pytest.mark.timeout(300)
-@pytest.mark.skip(reason="No meaningful dict.")
 def test_get_all() -> None:
     logger.info("Start of get_all() test.")
-    param_set = value_pool_fuzzer.fuzz(["DICT", "STRING"], 2)
+    param_set = value_pool_fuzzer.fuzz(["GRAMMAR_CONTROLS_JSON_COV", "STRING"], 2)
     result = param_runner.run(get_all, param_set)
```

Comment:
As expected, the testcase always fails, the expected strings in the `get_all()` function are never generated. Of course, it is also debatable whether the function `get_all()` in PyLoxone should not intercept this better.

Reply:
Good question. We may discuss this in the weekly. For now I think that's how we should test this function given the fuzzing approach.

Reply:
After [7505239](https://github.com/ThorbenCarl/PyLoxone/pull/18/commits/750523981a0fb60d1c638f5f280f27f8aaa67e5a) the test passes.